script/run/run.sh

```diff
@@ -43,7 +43,7 @@ if [ -n "$compose_dir" ]; then
     VOLUMES="$VOLUMES -v $compose_dir:$compose_dir"
 fi
 if [ -n "$HOME" ]; then
-    VOLUMES="$VOLUMES -v $HOME:$HOME -v $HOME:/root" # mount $HOME in /root to share docker.config
+    VOLUMES="$VOLUMES -v $HOME:$HOME -e HOME" # Pass in HOME to share docker.config and allow ~/-relative paths to work.
```

Comment:
You should be able to pass the `-e HOME` through the variable `COMPOSE_OPTIONS` (as described in line 11). If we remove this volume mount here, that can break peoples code that expects this mount.

Reply:
Yes, it can break code for anyone who reads files from /root when running docker-compose wrapper script as a non root user. That sounds like "working by accident", and works differently than the stand-alone binary.

I'd say this makes the whole wrapper script behave more like the stand-alone binary.

Reply:
Ok fair enough!